integration-tests/contract-originate-invalid-string.spec.ts

```diff
@@ -25,7 +26,11 @@ CONFIGS().forEach(({ lib, rpc, setup }) => {
           init: `"Copyright ©"`
         })
       } catch (ex) {
```

Comment:
Should be :
```
catch (ex) {
          expect(ex).toEqual(expect.objectContaining({ message: expect.stringContaining('non_printable_character') }))
      }
```

without distinction for protocol

Reply:
protocol removed